src/routes/_header.svelte

```diff
@@ -75,7 +75,7 @@
 
 <style lang="postcss">
   .header-button {
-    @apply block px-4 py-2 text-sm text-white flex items-center justify-center;
+    @apply px-4 py-2 text-sm text-white flex items-center justify-center;
```

Comment:
does the PR has a wrong title or hiding `temporal-system` namespace is missing?

Reply:
oh it's here https://github.com/temporalio/ui/pull/328/files#diff-e31925461f883c5fd62fb269b8572a4cbe2fb82350177e6263bba6037d537eeeR14